### PR TITLE
Merge master branch

### PR DESCRIPTION
**Pull request by aminya**
One of the PRs was using the wrong target branch

### CODE REVIEW

**Review by ddalcino — APPROVED** [review]
Good catch! I did not notice this.

As long as we're here, I noticed that there are still a couple of links here that point to `lefticus` rather than `cpp-best-practices`; we should probably move them.

File: README.md [hunk 1]
```diff
@@ -1,20 +1,16 @@
 # cpp_starter_project
 
-[![codecov](https://codecov.io/gh/lefticus/cpp_starter_project/branch/master/graph/badge.svg)](https://codecov.io/gh/lefticus/cpp_starter_project)
-
 [![Build Status](https://travis-ci.org/lefticus/cpp_starter_project.svg?branch=master)](https://travis-ci.org/lefticus/cpp_starter_project)
```
Comment (ddalcino) [inline README.md:3]: This would surely be a broken link if we linked to `cpp-best-practices`.

File: README.md [hunk 2]
```diff
@@ -1,20 +1,16 @@
...
-
 [![Build status](https://ci.appveyor.com/api/projects/status/ro4lbfoa7n0sy74c/branch/master?svg=true)](https://ci.appveyor.com/project/lefticus/cpp-starter-project/branch/master)
```
Comment (ddalcino) [inline README.md:4]:
```suggestion
[![Build status](https://ci.appveyor.com/api/projects/status/ro4lbfoa7n0sy74c/branch/master?svg=true)](https://ci.appveyor.com/project/cpp-best-practices/cpp-starter-project/branch/master)
```

File: README.md [hunk 3]
```diff
@@ -1,20 +1,16 @@
...
-
 ![CMake](https://github.com/lefticus/cpp_starter_project/workflows/CMake/badge.svg)
```
Comment (ddalcino) [inline README.md:5]:
```suggestion
![CMake](https://github.com/cpp-best-practices/cpp_starter_project/workflows/CMake/badge.svg)
```

File: README.md [hunk 4]
```diff
@@ -1,20 +1,16 @@
...
-
-[![Language grade: C++](https://img.shields.io/lgtm/grade/cpp/github/lefticus/cpp_starter_project)](https://lgtm.com/projects/g/lefticus/cpp_starter_project/context:cpp)
+[![codecov](https://codecov.io/gh/lefticus/cpp_starter_project/branch/master/graph/badge.svg)](https://codecov.io/gh/lefticus/cpp_starter_project)
```
Comment (ddalcino) [inline README.md:6]:
```suggestion
[![codecov](https://codecov.io/gh/cpp-best-practices/cpp_starter_project/branch/master/graph/badge.svg)](https://codecov.io/gh/cpp-best-practices/cpp_starter_project)
```

File: README.md (hunk 1 above)
Comment (lefticus) [reply README.md:3]: I'll check out codecov.io and appveyor for this move to the organization

**Review by lefticus — APPROVED** [review]
I think this is all good to go and correct now. Failing builds are due to lack of merge from main

**Comment by RalphSteinhagen** [comment]
@lefticus thanks for merging and sorry (mea culpa) for the PR towards 'master' and not 'main'. Didn't catch the transition (which is good). :+1:

**Pull request merged**